### PR TITLE
libzippp: add version 5.0

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **libzippp/5.0**

libzippp depends on zlib and libzip, and the versions of libzipp and libzip are strongly tied.

As follows.

| libzippp | libzip |
|-----------|------|
| 4.0-1.7.3  |  1.7.3 |
| 4.0-1.8.0  |  1.8.0 |
| 5.0-1.8.0  |  1.8.0 |

I don't think the version numbers of current recipes are accurate.
I think it should have been "4.0-1.7.3" instead of "4.0".
However, for compatibility, I'm going to add version "5.0" instead of  "5.0-1.8.0" as well.
If it's better to use the exact version, I'll fix it.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/libzippp/all/conandata.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 sources:
+  "5.0":
```
Comment (madebr) [inline recipes/libzippp/all/conandata.yml:2]: I agree with you about the versioning.
`5.0-1.8.0` is the real version.

Comment (toge) [reply recipes/libzippp/all/conandata.yml]: I changed the version name from `5.0` to `5.0-1.8.0`.

**Review by madebr — DISMISSED** [review]

File: recipes/libzippp/all/conanfile.py [hunk 2]
```diff
@@ -37,14 +38,15 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("libzip/1.7.3")
+        self.requires("zlib/1.2.11")
+        if tools.Version(self.version) == "4.0":
```
[the remark this answers is not in the document]
Comment (toge) [reply recipes/libzippp/all/conanfile.py:47]: @ericLemanissier 
Sorry for late.
I understand your point is to check libzip version which may be overwritten by other recipes.
If it is true, I wrote following code in `validate(self)` to solve it:

```python
    libzip_version = "1.7.3"
    if libzippp_version != "4.0":
        libzip_version = libzippp_version.split("-")[1]
    if self.deps_cpp_info["libzip"].version != libzip_version:
        raise tools.ConanInvalidConfiguration("{}: version of libzip must be {}".format(self.name, libzip_version))
```

But in `validate(self)`, deps_cpp_info doesn't have a `libzip` key.
Do you know the way to get libzip version in `validate(self)`?

[the remark this answers is not in the document]
Comment (ericLemanissier) [reply recipes/libzippp/all/conanfile.py:47]: Ow, maybe this information is not available yet. Ok!

File: recipes/libzippp/all/conanfile.py [hunk 3]
```diff
@@ -75,7 +83,6 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = tools.collect_libs(self)
-        self.cpp_info.names["cmake_find_package"] = "libzippp"
-        self.cpp_info.names["cmake_find_package_multi"] = "libzippp"
+        self.cpp_info.set_property("cmake_file_name", "libzippp")
```
Comment (ericLemanissier) [inline recipes/libzippp/all/conanfile.py:86]: Please restore the 2 lines above

**Review by ericLemanissier — CHANGES_REQUESTED** [review]

File: recipes/libzippp/all/conanfile.py (hunk 3 above)
Comment (toge) [reply recipes/libzippp/all/conanfile.py:88]: Sorry. I forget to restore. 
i fixed it.

**Review by madebr — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**